cake.cs

```diff
@@ -1,16 +1,15 @@
+#:sdk Cake.Sdk
+#:package Cake.FileHelpers
+#:package Newtonsoft.Json
```

Comment:
This is great if it ties in with Directory.Packages.props as it means we'll get automated PRs to keep them up to date.

Reply:
Yes, it ties in with CPM if a `Directory.Packages.prop` exists.